finals-2024-01-16/code/5159-ana-ilievska/src/app/models/mapping.ts

```diff
@@ -0,0 +1,49 @@
+import { Album, Band, PlaceData } from './client';
+import { response } from "express"
+import { AlbumResponse, BandResponse, PlaceDataResponse } from "./server"
+
+export const toBand = (response: BandResponse): Band => {
```

Comment:
The point of the mapping is to transform the data to a better format. In this case both formats are the same, so the transformation is in place. Still, the code is ok, +5 points